scripts/_helpers.py

```diff
@@ -107,11 +107,26 @@ def get_run_path(fn, dir, rdir, shared_resources, exclude_from_shared):
         irrelevant_wildcards = {"technology", "year", "scope", "kind"}
         no_relevant_wildcards = not existing_wildcards - irrelevant_wildcards
         not_shared_rule = (
-            not fn.startswith("networks/elec")
+            not fn.endswith("elec.nc")
             and not fn.startswith("add_electricity")
             and not any(fn.startswith(ex) for ex in exclude_from_shared)
         )
         is_shared = no_relevant_wildcards and not_shared_rule
+        shared_files = (
+            "networks/base_s_{clusters}.nc",
+            "regions_onshore_base_s_{clusters}.geojson",
+            "regions_offshore_base_s_{clusters}.geojson",
+            "busmap_base_s_{clusters}.csv",
+            "linemap_base_s_{clusters}.csv",
+            "cluster_network_base_s_{clusters}",
+            "profile_{clusters}_",
+            "build_renewable_profile_{clusters}",
+            "availability_matrix_",
+            "determine_availability_matrix_",
+            "solar_thermal",
+        )
+        if any(prefix in fn for prefix in shared_files) or is_shared:
```

Comment:
why is the `or is_shared` necessary? Looks like it could be omitted

Reply:
In my opinion, this is necessary here.
Because for the first `is_shared`, in the previous behaviour, you had neither a relevant_wildcard nor a shared_rule to make it shared. However, with the new structure, this would always be False for the profiles, meaning that they are never shared in the end.
This is why I added `shared_files`, and if any of those is in the file name, then the resource is always shared.